cookbooks/convection-box/doc/convection-box.md

```diff
@@ -168,28 +174,29 @@ the same.
 ##### Visualizing results.
 
 Having run the program, we now want to visualize the numerical results we got.
-can generate graphical output in formats understood by pretty much any
+ASPECT can generate graphical output in formats understood by pretty much any
 visualization program (see the parameters described in
-Section&nbsp;[\[parameters:Postprocess/Visualization\]][4]) but we will here
-follow the discussion in Section&nbsp;[\[sec:viz\]][5] and use the default VTU
+Section&nbsp;{ref}`parameters:Postprocess/Visualization`) but we will here
+follow the discussion in Section&nbsp;{ref}`sec:viz` and use the default VTU
 output format to visualize using the Visit program.
 
 In the parameter file we have specified that graphical output should be
 generated every 0.01 time units. Looking through these output files (which can
 be found in the folder `output-convection-box`, as specified in the input
 file), we find that the flow and temperature fields quickly converge to a
-stationary state. Fig.&nbsp;[2][] shows the initial and final states of this
-simulation.
+stationary state. {numref}`fig:convection-box-fields` shows the initial
+and final states of this simulation.
 
-.
-<img src="cookbooks/convection-box/doc/visit0000.png" title="fig:" id="fig:convection-box-fields" style="width:40.0%" alt="Convection in a box: Initial temperature and velocity field (left) and final state (right)." />
-<img src="cookbooks/convection-box/doc/visit0001.png" title="fig:" id="fig:convection-box-fields" style="width:40.0%" alt="Convection in a box: Initial temperature and velocity field (left) and final state (right)." />
-.
+```{figure-md} fig:convection-box-fields
+<img src="convection_box_fields.*" style="width:80.0%" />
+
+Convection in a box: Initial temperature and velocity field (left) and final state (right).
+```
 
 There are many other things we can learn from the output files generated by ,
 specifically from the statistics file that contains information collected at
 every time step and that has been discussed in
-Section&nbsp;[\[sec:viz-stat\]][6]. In particular, in our input file, we have
+Section&nbsp;{ref}`sec:viz-stat`. In particular, in our input file, we have
```

Comment:
Please replace Section&nbsp;{ref}`sec:viz-stat` with {ref}`sec:run-aspect:visualize:stat-data`

(This section isn't technically setup yet so that reference tag is brand new, but it will be merged soon)

Reply:
This appears on line 232 as well